### DISCUSSION TITLE
dbt + ML

### DISCUSSION THREAD

**Discussion by gwenwindflower** [dbt-labs/docs.getdbt.com#1158] [Archive]
We should create some foundational content around using dbt models to provide features to ML models. There are a few approaches we'll want to cover here including:

- tool agnostic basics
- working with Continual.ai
- working with fal
- others ?

**Comment by johnblust** [comment]
Hey @burkaygur, this discussion is referencing the previous Issue you opened in the old repo ([linked here](https://github.com/dbt-labs/dbt-technical-blog-writing/issues/88)). Is this something you are still interested in working on together? Happy to help get this moving any way I can.

**Comment by burkaygur** [comment]
Hi @johnblust! Yes, would like to work together on this! We have a draft going that is ~80% done. What would be the best way to share that with you?

**Reply by johnblust** [reply]
Awesome! Glad to see you're still working on it :) Going to be working on this with @amychen1776. We'll let you know next steps end of next week/beginning of the week of April 4th as they are OOO until then. Talk to you soon!
